.github/workflows/subgraph.yml

```diff
@@ -6,7 +6,6 @@ on:
       - main
 
 jobs:
-
```

Comment:
Doesn't you need to install the dependencies to run the tests?

Reply:
It does install the dependencies at line 30